<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleTrack</class>
 <widget class="QMainWindow" name="singleTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>单目标追踪</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,0">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelShow">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>600</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>900</width>
           <height>800</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="processShow"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="openCamera">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>打开相机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openVideo">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>打开视频</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectObj">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>选择跟踪目标</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startTrack">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>开始跟踪</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="endTrack">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>结束跟踪</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
